src/program.js

```diff
@@ -601,6 +601,11 @@ Example: $0 --help run.
             'Only used with `use-submission-api`',
           type: 'string',
         },
+        'version-source': {
+          describe:
+            'Path to a zip file containing human readable source code for a version. ' +
```

Comment:
```suggestion
            'Path to a zip file containing human readable source code of this submission. ' +
```

"a version" is vague, how about this? Other suggestions are welcome.

Ideally the documentation should be clear in expressing the requirement that the submitted add-on can be built from the zip, e.g. by linking to the extension workshop article that documents the requirements.

Reply:
EW is here: https://extensionworkshop.com/documentation/publish/source-code-submission/

Reply:
reworded